lib/shopify_api/resources/draft_order.rb

```diff
@@ -0,0 +1,10 @@
+module ShopifyAPI
+  class DraftOrder < Base
+    include Metafields
+
+    def send_invoice(draft_order_invoice = ShopifyAPI::DraftOrderInvoice.new)
+      response = post(:send_invoice, {}, draft_order_invoice.encode)
+      load_attributes_from_response(response)
```

Comment:
Shouldn't this be `instantiate_record(format.decode(response.body), {})` so you'll get an `DraftOrderInvoice` back instead of putting the attributes on `self` (i.e. the `DraftOrder`)